libraries/tools/kotlin-gradle-plugin/src/testData/kotlin.incremental.useClasspathSnapshot/changedMethodImplementation/com/example/A.kt

```diff
@@ -0,0 +1,12 @@
+package com.example
+
+class A {
```

Comment:
Can you please add more test cases:
- private/non-private fields
- inline functions
- changing supertype by adding somethings that changes/does not change the supertype ABI
- adding an annotation

we do not need to test every edge case (there are a lot of incremental tests), but this is just to leverage unit tests as much as possible.

Reply:
Yup, I've added a TODO here and will come back to it once we have a working version.